tools/biotransformer/biotransformer.xml

```diff
@@ -51,43 +38,46 @@
     <param name="tolerance" type="float" value="0.01" label="Mass tolerance for metabolite identification."/>
 
     <conditional name="input">
-    	<param name="type_selector" type="select" label="Input format [SMILES, SDF, CSV].">
-        	<option value="smile" selected="true">SMILES</option>
-		<option value="sdf">SDF</option>
-		<option value="csv">CSV</option>
+    	<param name="type_selector" type="select" label="Input format [CSV].">
+		<option value="csv" selected="true">CSV</option>
```

Comment:
Given there is no other option to choose from we can drop the whole selector I think.

Reply:
I left it there because in future I plan to add input type of simple SMILES string, but it is not a priority now.

Reply:
For the moment this is a dead code and I think we should remove it. You can always go back to this version using git when you decide to expand the implementation.